MACSio.spec

```diff
@@ -14,7 +14,7 @@ BuildRequires:  cmake
 BuildRequires:  json-cwx
 BuildRequires:  hdf5-devel
 BuildRequires:  hdf5-mpich-devel
-BuildRequires:  mpich
+BuildRequires:  mpich-devel
```

Comment:
Indeed.  You should always do a double-take when you see a BR: that is not a `-devel`.  It's certainly not a hard and fast rule, but if it's for a library, it should almost always be a `-devel`.